test_vm/tests/verified_claim_test.rs

```diff
@@ -428,3 +428,133 @@ fn expired_allocations_test<BS: Blockstore>(v: &dyn VM<BS>) {
 
     expect_invariants(v, &[invariant_failure_patterns::REWARD_STATE_EPOCH_MISMATCH.to_owned()]);
 }
+
+#[test]
+fn deal_passes_claim_fails() {
+    let store = MemoryBlockstore::new();
+    let v = TestVM::<MemoryBlockstore>::new_with_singletons(&store);
+
+    // setup miner, verifreg client, make deals, prove sector
```

Comment:
This comment isn't super helpful -- consider removing, or changing to "try to prove sectors"